### PR TITLE
feat: add more zsh configs

### PR DESCRIPTION
**Pull request by bayou-brogrammer**

### CODE REVIEW

File: dx/usr/etc/zsh/zshrc [hunk 1]
```diff
@@ -1,38 +1,208 @@
...
-## restart running processes on exit
-#setopt HUP
+# Standard style used by default for 'list-colors'
+LS_COLORS=${LS_COLORS:-'di=34:ln=35:so=32:pi=33:ex=31:bd=36;01:cd=33;01:su=31;40;07:sg=36;40;07:tw=32;40;07:ow=33;40;07:'}
+
+# Set name of the theme to load --- if set to "random", it will
+# load a random theme each time oh-my-zsh is loaded, in which case,
+# to know which specific one was loaded, run: echo $RANDOM_THEME
+# See https://github.com/ohmyzsh/ohmyzsh/wiki/Themes
+ZSH_THEME="robbyrussell"
```
[the remark this answers is not in the document]
Comment (bayou-brogrammer) [reply dx/usr/etc/zsh/zshrc]: ZSH_THEME has its usecases outside of ohmyzsh, but you are correct. I defaulted it to starship

**Comment by bketelsen** [comment]
@rothgar can you take a look?

File: dx/usr/etc/zsh/zlogin [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env zsh
+# vim:syntax=zsh
+# vim:filetype=zsh
+
+# Execute code in the background to not affect the current session
+(
+    # <https://github.com/zimfw/zimfw/blob/master/login_init.zsh>
+    setopt LOCAL_OPTIONS EXTENDED_GLOB
+    autoload -U zrecompile
+    local ZSHCONFIG=~/.zsh
```
Comment (rothgar) [inline dx/usr/etc/zsh/zlogin:10]: Is there a reason this wouldn’t use ZDOTDIR?

File: dx/usr/etc/zsh/zshrc [hunk 3]
```diff
@@ -1,38 +1,214 @@
...
+# Standard style used by default for 'list-colors'
+LS_COLORS=${LS_COLORS:-'di=34:ln=35:so=32:pi=33:ex=31:bd=36;01:cd=33;01:su=31;40;07:sg=36;40;07:tw=32;40;07:ow=33;40;07:'}
+
+#
+# Styles
+#
+
+# zstyle ':omz:update' mode disabled  # disable automatic updates
+# zstyle ':omz:update' mode auto      # update automatically without asking
+zstyle ':omz:update' mode reminder # just remind me to update when it's time
```
Comment (rothgar) [inline dx/usr/etc/zsh/zshrc:67]: Do we use/want oh-my-zsh settings. I don’t _think_ they affect anything if it’s not installed but don’t think it’s needed.

File: dx/usr/etc/zsh/zshrc [hunk 4]
```diff
@@ -1,38 +1,214 @@
...
\ No newline at end of file
+colors
+
+#####################
+# Sourcing
+#####################
+
+# Brew
+if [[ -o interactive ]] && [[ -d /home/linuxbrew/.linuxbrew ]]; then
+eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
```
Comment (rothgar) [inline dx/usr/etc/zsh/zshrc:196]:
```suggestion
    eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
```

**Comment by rothgar** [comment]
This may also be helpful with brew available on the host
```
# load brew autocomplete
if [ -d "/home/linuxbrew/.linuxbrew/share/zsh/site-functions" ]; then
    fpath+=(/home/linuxbrew/.linuxbrew/share/zsh/site-functions)
fi
```

**Review by castrojo — APPROVED** [review]

**Comment by m2Giles** [comment]
This looks awesome.

**Pull request merged**